gnosis/cowswap
# [1053] Re-enable common bases

Closes #1053 

# Summary

Re-enables common tokens known as common bases in Uni. It shows to the user an opinionated list of common tokens used to trade above the token list for easy of token selection.

![image](https://user-images.githubusercontent.com/21335563/126993015-8ea41745-24b4-47f1-8b39-2aec0cda30bd.png)

  # To Test

1. open the currency picker
2. make sure the hover actions / colours look good
3. select a token and make sure that for the same input type the selected token, if in the common tokens list, is displayed as disabled
